Source/ACE/Entity/GenericObject.cs

```diff
@@ -15,46 +15,24 @@ public GenericObject(AceObject aceObject)
         {
         }
 
-        public GenericObject(ObjectGuid guid, AceObject aceObject)
-            : base(guid, aceObject)
-        {
-        }
+        ////public GenericObject(ObjectGuid guid, AceObject aceObject)
```

Comment:
At this point the class serves no purpose.  It's a commented out class with a constructor.  Just remove it.